source/wp-content/themes/wporg-main-2022/functions.php

```diff
@@ -323,3 +324,13 @@ function update_header_template_part_class( $block ) {
 	}
 	return $block;
 }
+
+/**
+ * Change document title separator
+ *
+ * @param string $title
+ * @return string
+ */
+function document_title_separator( $title ) {
```

Comment:
```suggestion
 * @param string $sep
 *
 * @return string
 */
function document_title_separator( $sep ) {
```

Minor formatting & param name suggestion (`$sep` comes from the name [in core](https://developer.wordpress.org/reference/hooks/document_title_separator/)).